tests/kernel/smp/testcase.yaml

```diff
@@ -1,3 +1,5 @@
 tests:
   kernel.multiprocessing.smp:
     filter: (CONFIG_MP_NUM_CPUS > 1)
+    extra_configs:
+      - CONFIG_TRACE_SCHED_IPI=y
```

Comment:
Put this into prj.conf.  You use extra_configs only when you have lots of sub-tests in a single YAML file and they need to share a base configuration with a few differences.

Reply:
Ok, put this CONFIG into the prj.conf.